docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   jekyll:
-    command: bash -c "apt-get update -y && apt-get install -y cmake pkg-config && bundle install && bundle exec jekyll serve -s jekyll --incremental --livereload --host=0.0.0.0"
+    command: bash -c "apt-get update -y && apt-get install -y cmake pkg-config && bundle install && JEKYLL_MINIBUNDLE_MODE=development bundle exec jekyll serve -s jekyll --incremental --host=0.0.0.0"
```

Comment:
This prevents the fingerprinting of the assets in development which breaks when we use `--incremental`:  https://github.com/tkareine/jekyll-minibundle#asset-fingerprinting

Removed `--livereload` because I had inconsistencies with this new workflow. Will see if we can re-enable it in the future